kobo/settings/base.py

```diff
@@ -288,7 +288,8 @@
         ]),
         # The available fields are hard-coded in the front end
         'Display (and optionally require) these metadata fields for users. '
-        "Possible fields are 'full_name', 'organization', 'organization_website', "
+        "Possible fields are:\n"
+        " 'full_name', 'organization', 'organization_website', "
         "'sector', 'gender', 'bio', 'city', 'country', 'twitter', 'linkedin', "
         "and 'instagram'.\n\n"
         "To add another language, use 'some-other-language' as an example.",
```

Comment:
This description seems wrong. I guess it should be the same as `PROJECT_METADATA_FIELDS`?